datastore/providers/azurecosmosdb_datastore.py

```diff
@@ -27,6 +27,8 @@
 AZCOSMOS_CONNSTR = os.environ.get("AZCOSMOS_CONNSTR")
 AZCOSMOS_DATABASE_NAME = os.environ.get("AZCOSMOS_DATABASE_NAME")
 AZCOSMOS_CONTAINER_NAME = os.environ.get("AZCOSMOS_CONTAINER_NAME")
+AZCOSMOS_SIMILARITY = os.environ.get("AZCOSMOS_SIMILARITY", "COS")
```

Comment:
Would you mind updating the table in https://github.com/openai/chatgpt-retrieval-plugin/blob/b02139d174e2c2b3de6a17ed4914c05f4a552c2f/docs/providers/azurecosmosdb/setup.md?plain=1#L17 as well?

Reply:
Updated! Thanks